.github/workflows/packages-upload-images.yml

```diff
@@ -0,0 +1,224 @@
+              - 'packages/debs/arm64/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_deb_agent_builder_armhf:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/debs/armhf/agent/**'
+              - 'packages/debs/utils/**'
+            pkg_rpm_agent_builder_arm64:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/arm64/agent/**'
+              - 'packages/rpms/utils/**'
+            pkg_rpm_agent_builder_armhf:
+              - 'packages/build.sh'
+              - 'packages/generate_package.sh'
+              - 'packages/rpms/armhf/agent/**'
+              - 'packages/rpms/utils/**'       
+            compile_windows_agent:
+              - 'packages/windows/**'
+            commom_wpk_builder:
+              - packages/wpk/wpkpack.py
+              - packages/wpk/run.sh
+              - packages/wpk/generate_wpk_package.sh
+              - packages/wpk/common/*
+            linux_wpk_builder_x86_64: 
+              - packages/wpk/wpkpack.py
+              - packages/wpk/run.sh
+              - packages/wpk/generate_wpk_package.sh
+              - packages/wpk/linux/x86_64/*
+
+      - name: Set TAG and WAZUH_AGENT_PACKAGES_BRANCH
+        run: |
+          VERSION=$(sed 's/^v\([0-9]*\.[0-9]*\.[0-9]*\)/\1/' src/VERSION)
+          if [ "${{ inputs.docker_image_tag }}" == "auto" ]; then echo "TAG=$VERSION" >> $GITHUB_ENV;
+          elif [ "${{ inputs.docker_image_tag }}" == "developer" ]; then echo "TAG=${{ github.ref_name }}" >> $GITHUB_ENV;
+          else echo "TAG=${{ inputs.docker_image_tag }}" >> $GITHUB_ENV; fi
+          echo "WAZUH_AGENT_PACKAGES_BRANCH=master" >> $GITHUB_ENV
+
+      - name: Request pkg_deb_manager_builder_amd64 update 
+        if: steps.changes.outputs.pkg_deb_manager_builder_amd64 == 'true'
+        run: |
+          gh workflow run packages-upload-manager-images.yml -r ${{ github.ref_name }} -f docker_image_tag=${{ env.TAG }} -f system=deb -f source_reference=${{ github.ref_name }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Request pkg_rpm_manager_builder_amd64 update
+        if: steps.changes.outputs.pkg_rpm_manager_builder_amd64 == 'true'
+        run: |
+          gh workflow run packages-upload-manager-images.yml -r ${{ github.ref_name }} -f docker_image_tag=${{ env.TAG }} -f system=rpm -f source_reference=${{ github.ref_name }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Request pkg_deb_agent_builder_amd64 update
+        if: steps.changes.outputs.pkg_deb_agent_builder_amd64 == 'true'
+        run: |
+          gh workflow run packages-upload-agent-images-amd.yml --repo wazuh/wazuh-agent-packages -r ${{ env.WAZUH_AGENT_PACKAGES_BRANCH }} -f docker_image_tag=${{ env.TAG }} -f system=deb -f architecture=amd64 -f legacy=false -f source_reference=${{ github.ref_name }}
```

Comment:
I'm not sure about this, but if a flag like *legacy* defaults to false and isn't required, is it still necessary to define it here? If not, I'd like to remove it.

Reply:
Consider applying this approach to all cases.